test.json

```diff
@@ -1,145 +1,47 @@
 [
-  {
-    "name": "fs-extra",
-    "repo": "https://github.com/jprichardson/node-fs-extra",
-    "description": "fs-extra contains methods that aren't included in the vanilla Node.js fs package. Such as mkdir -p, cp -r, and rm -rf.",
-    "dependents": 3279,
-    "disable": true
-  },
   {
     "name": "tap",
     "repo": "https://github.com/tapjs/node-tap",
     "description": "A Test-Anything-Protocol library",
     "dependents": 3050,
     "args": ["lib", "test"]
   },
-  {
-    "name": "yargs",
-    "repo": "https://github.com/yargs/yargs",
-    "description": "Light-weight option parsing with an argv hash. No optstrings attached.",
-    "dependents": 2526,
-    "disable": true
-  },
-  {
-    "name": "karma-coverage",
-    "repo": "https://github.com/karma-runner/karma-coverage",
-    "description": "A Karma plugin. Generate code coverage.",
-    "dependents": 1743,
-    "disable": true
-  },
-  {
-    "name": "semantic-release",
-    "repo": "https://github.com/semantic-release/semantic-release",
-    "description": "automated semver compliant package publishing",
-    "dependents": 771,
-    "disable": true
-  },
   {
     "name": "karma-cli",
     "repo": "https://github.com/karma-runner/karma-cli",
     "description": "The Karma command line interface.",
     "dependents": 517
   },
-  {
-    "name": "lru-cache",
-    "repo": "https://github.com/isaacs/node-lru-cache",
-    "description": "A cache object that deletes the least-recently-used items.",
-    "dependents": 413,
-    "disable": true
-  },
   {
     "name": "co-mocha",
     "repo": "https://github.com/blakeembrey/co-mocha",
     "description": "Enable support for generators in Mocha tests",
     "dependents": 388
   },
-  {
-    "name": "dotenv",
-    "repo": "https://github.com/motdotla/dotenv",
-    "description": "Loads environment variables from .env file",
-    "dependents": 377,
-    "disable": true
```

Comment:
Let's keep this one. It now passes, plus they're on standard 13 and they seem to be keeping up to date (14 only came out a month ago).